packages/ckeditor5-ui/src/toolbar/toolbarview.js

```diff
@@ -277,6 +288,22 @@ export default class ToolbarView extends View {
 		this.items.addMany( config.map( name => {
 			if ( name == '|' ) {
 				return new ToolbarSeparatorView();
+			} else if ( name == '-' ) {
+				if ( this.options.shouldGroupWhenFull ) {
+					/**
+					 * Toolbar line break will work only when button grouping is disabled in the toolbar config:
+					 *
+					 * 		Editor.defaultConfig = {
+					 * 			toolbar: {
+					 * 				shouldNotGroupWhenFull: true
```

Comment:
It may be unclear whether this error shows up because of this option or the option is a remedy. This should be either explained before the snippet or as an inline comment in the code.